cmd/launcher/launcher.go

```diff
@@ -288,6 +288,19 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		if err := controlService.RegisterConsumer(notificationconsumer.NotificationSubsystem, notificationConsumer); err != nil {
 			return fmt.Errorf("failed to register notify consumer: %w", err)
 		}
+
+		// Set up our tracing instrumentation
+		if exp, err := exporter.NewTraceExporter(ctx, k, extension, logger); err != nil {
+			level.Debug(logger).Log(
+				"msg", "could not set up trace exporter",
+				"err", err,
+			)
+		} else {
+			runGroup.Add(exp.Execute, exp.Interrupt)
+			if err := controlService.RegisterConsumer(exporter.IngestSubsystem, exp); err != nil {
```

Comment:
TODO:
* New consumer for ingest subsystem
* Make exporter a subscriber of the ingest subsystem (lets @James-Pickett 's work in https://github.com/kolide/launcher/pull/1228 also subscribe to the ingest subsystem)
* Update diagram